kommandir/bin/adept_openstack.py

```diff
@@ -158,9 +158,9 @@ def __init__(self, logging_level, verbose):
         self.verbose = verbose
 
     def filter(self, record):
-        # record.msg may not be string; use getMessage to guarantee string form
-        msg = record.getMessage()
-        if msg.startswith('>'):
+        # record.msg may not be string, but '>' processing requires one
+        msg = record.msg
+        if isinstance(msg, str) and msg.startswith('>'):
```

Comment:
Ya, this is essentially what I tried...but I won't rule out my having made some typo...